contracts/eosio.bios/src/eosio.bios.cpp

```diff
@@ -20,6 +20,21 @@ void bios::setabi( name account, const std::vector<char>& abi ) {
    }
 }
 
+// helpers for defining std::unordered_set<eosio::bls_g1>
+struct g1_hash {
+   // bls_g1 is defined as std::array<char, 96>
+   std::size_t operator()(const eosio::bls_g1& g1) const {
+      std::hash<std::string> hash_func;
```

Comment:
I think you could use `std::hash<const char*>` instead as each memory location will be unique and should be quicker than hashing all 96 bytes. 

> There is no specialization for C strings. std::hash<const char*> produces a hash of the value of the pointer (the memory address), it does not examine the contents of any character array.